releasenotes/notes/preset-pass-manager-approximation-degree-503198b1974adf79.yaml

```diff
@@ -0,0 +1,12 @@
+---
+upgrade:
+  - |
+    The implicit use of ``approximation_degree!=1.0`` by default in the
```

Comment:
FWIW, I think was only the case if a backend or target with error properties are set. The default of `approximation_degree=None` was a short hand of approximate up to the error rate reported in the target. So if there was no error rate it wouldn't approximate the decomposition.

Reply:
tbh I just copied the first sentence of the note from #8595.  Happy to change it to whatever you like

Reply:
It's fine this was more just me musing out loud. It's why I didn't leave a concrete suggestion.